webui-src/app/mail/mail_util.js

```diff
@@ -148,18 +160,55 @@ const Table = () => {
           m('th[title=starred]', m('i.fas.fa-star')),
           m('th[title=attachments]', m('i.fas.fa-paperclip')),
           m('th', 'Subject'),
-          // m('th', 'From'),
+          m('th', 'From'),
           m('th', 'Date'),
         ]),
         v.children,
       ]),
   };
 };
-
+const SearchBar = () => {
+  let searchString = '';
+  return {
+    view: (v) =>
+      m('input[type=text][id=searchmail][placeholder=Search Subject].searchbar', {
+        value: searchString,
+        oninput: (e) => {
+          searchString = e.target.value.toLowerCase();
+          for (const hash in v.attrs.list) {
+            if (v.attrs.list[hash].fname.toLowerCase().indexOf(searchString) > -1) {
+              v.attrs.list[hash].isSearched = true;
+            } else {
+              v.attrs.list[hash].isSearched = false;
+            }
+          }
+        },
+      }),
+  };
+};
+function popupMessageCompose(message) {
+  const container = document.getElementById('modal-container');
+  container.style.display = 'block';
+  m.render(
+    container,
```

Comment:
Hmm, I'm not sure about this.
It's fine for now, but since this is rendered outside of the main mithril loop, any elements we add to this will not be out of mithril's control.
I feel this might cause issues in the future when we add more elements to messages.